server/command/user_leave_rotation.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package command
+
+import (
+	"github.com/mattermost/mattermost-plugin-solar-lottery/server/sl"
+	"github.com/mattermost/mattermost-plugin-solar-lottery/server/utils/md"
+)
+
+func (c *Command) leaveRotation(parameters []string) (md.MD, error) {
+	c.withFlagRotation()
```

Comment:
It's difficult because it is special (positional/flag combination).

A question I would like to ask (myself) is why is the rotation name/ID convention is different from say, the tasks. I have no good answer to that, yet. Perhaps need to either stick with IDs everywhere, no names (that'd be not user-friendly after the first "archive"), or come up with a better "unique active name" resolution (and flags) that works for all.

Reply:
Inclined not to fix this, yet - another time.